definitions/ext-aerospike/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: AEROSPIKE
+
+synthesis:
+  identifier: targetName
```

Comment:
Just wanted to double check: Would this identifier be unique within an account ? Or could there be two aerospike entities with the same targetName ?

Reply:
As per the current requirement, we are only planning for one entity of aerospike.

Reply:
Hi @andre-nr , I have updated the synthesis identifier and the name. Request you to kindly review it.